tests/lax_numpy_test.py

```diff
@@ -2154,6 +2154,24 @@ def testSplitStaticInt(self, shape, num_sections, axis, dtype, rng_factory):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_axis={}_{}sections".format(
+          jtu.format_shape_dtype_string(shape, dtype), axis, num_sections),
+       "shape": shape, "num_sections": num_sections, "axis": axis,
+       "dtype": dtype, "rng_factory": jtu.rand_default}
```

Comment:
We no longer pass rng_factory as a testcase argument. You can use `jtu.rand_default` directly.